src/extension.ts

```diff
@@ -144,6 +151,18 @@ export async function activate(context: vscode.ExtensionContext) {
                 await showQuickStartWebview(context)
             }
         })
+        registerCommand({
```

Comment:
These would go into the register commands portion of the cdk's activation file to keep a feature's activation code together

Reply:
Makes sense. I get the reorg of activation a little better now :)
updating!